src/main/java/org/sagebionetworks/web/client/jsinterop/AccessRequirementListProps.java

```diff
@@ -18,18 +19,21 @@ public interface Callback {
   boolean renderAsModal;
   Object[] accessRequirementFromProps;
   Callback onHide;
-  String entityId;
+  String subjectId;
+  String subjectType;
 
   @JsOverlay
   public static AccessRequirementListProps create(
     Callback onHide,
     List<AccessRequirement> accessRequirements,
-    String entityId
+    String subjectId,
+    RestrictableObjectType subjectType
   ) {
     AccessRequirementListProps props = new AccessRequirementListProps();
     props.renderAsModal = true;
     props.onHide = onHide;
-    props.entityId = entityId;
```

Comment:
Use subject Id and RestrictableObjectType enumeration value in AccessRequirementList instead of deprecated entityId prop value.